web/server/entrypoint.go

```diff
@@ -150,7 +150,7 @@ func RunServer(ctx context.Context, args []string, _ logging.Logger) (err error)
 				ID:         cfgFromDisk.Cloud.ID,
 				Secret:     cfgFromDisk.Cloud.Secret,
 			},
-			nil, false, logger.Sublogger("networking").Sublogger("netlogger"),
+			nil, false, logger.Sublogger("netlogger"),
```

Comment:
Not a mandate, but `logging.NewLogger("rdk.networking.netlogger")` is also suitable here if we wanted to keep the "name".

Reply:
Ah good point; did `logger.Sublogger("networking.netlogger")` instead. Same idea, right?

Reply:
I think that's obscuring the intent here. Use `logging.NewLogger` is clear that:
* We're not going into the registry
* We're not "borrowing" anything from the `logger` state.

Also, I think `logger.Sublogger("networking.netlogger")` is relying on an implementation detail. Specifically, `Sublogger` could be implemented as:
* For each "dot-delimited piece" in the sublogger name
  * Get or create that piece

Meaning we'd turn `logger.Sublogger("networking.netlogger")` internally into two calls:
```
logger = logger.Sublogger("networking")
logger = logger.Sublogger("netlogger")
return logger
```
Which would reintroduce the bug.

Reply:
Yepp you're totally right thanks for spelling that out for me 😮‍💨 . Opted for `logging.NewLogger("rdk.networking.netlogger")` as you originally suggested.